core/unix/signal_private.h

```diff
@@ -191,7 +191,7 @@ typedef _STRUCT_UCONTEXT /* == __darwin_ucontext */ kernel_ucontext_t;
  * (these are from /usr/src/linux/arch/i386/kernel/signal.c for kernel 2.4.17)
```

Comment:
Probably "pp_include" only matches "#include" with no space and you get two tokens for "# include".  What I did when developing the existing vera rules was print out all the tokens in a file to understand what their lexer is doing.